From 4a5b945de8f67b24144078b1be6dfe79074153fe Mon Sep 17 00:00:00 2001
From: ChenXiaoSong <chenxiaosong@kylinos.cn>
Date: Wed, 19 Aug 2026 07:12:24 +0000
Subject: [PATCH 15118/15333] MAINTAINERS: add myself as KSMBD reviewer

I and my team have been working on KSMBD development, and I am also
interested in helping maintain this code.

Signed-off-by: ChenXiaoSong <chenxiaosong@kylinos.cn>
Signed-off-by: Namjae Jeon <linkinjeon@kernel.org>
---
 MAINTAINERS | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/MAINTAINERS b/MAINTAINERS
index 8014b9f8253e..a0e04c0ba453 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -14138,6 +14138,8 @@ M:	Steve French <smfrench@gmail.com>
 M:	Steve French <sfrench@samba.org>
 R:	Sergey Senozhatsky <senozhatsky@chromium.org>
 R:	Tom Talpey <tom@talpey.com>
+R:	ChenXiaoSong <chenxiaosong@chenxiaosong.com>
+R:	ChenXiaoSong <chenxiaosong@kylinos.cn>
 L:	linux-cifs@vger.kernel.org
 S:	Maintained
 T:	git https://git.samba.org/ksmbd.git ksmbd-for-next
-- 
2.55.0

